src/LruArrayCache.php

```diff
@@ -49,6 +50,7 @@ public function get($key)
 
     public function set($key, $value, $ttl = 0)
     {
+        $key = $key ?? '';
```

Comment:
The same here.  I don`t like the idea of making the key an empty string. I think we could either fail because the key is null or do an early return.